pkgs/development/tools/misc/libtool/libtool2.nix

```diff
@@ -12,7 +12,7 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "lib" ];
 
-  nativeBuildInputs = [ perl help2man ];
+  nativeBuildInputs = [ perl help2man m4 ];
```

Comment:
Is it really both `nativeBuildInput` and `propagatedBuildInput`?

Reply:
It's certainly possible, and fairly harmless if it isn't. But we can always try without too.